taskiq/abc/result_backend.py

```diff
@@ -50,3 +60,25 @@ async def get_result(
         :param with_logs: if True it will download task's logs.
         :return: task's return value.
         """
+
+    async def set_progress(
+        self,
+        task_id: str,
+        progress: "TaskProgress[_ProgressType]",
```

Comment:
Just set generic parameter to Any, because ResultBackend doesn't need to know about exact type. And we cannot parametrize it with this type, since it can vary from task to task.

```suggestion
        progress: "TaskProgress[Any]",
```

Reply:
Will be easy to switch to PEP696

Reply:
ResultBackend does't know about it right now, but then PEP696 is implemented it will know

Reply:
I wanna add second generic parameter in ResultBackend 
But for background compatibility it should have default value

Reply:
We don't need to switch to this pep for result backend. My point is that we don't need to set it as generic value, because it's not used in any place. This generic doesn't have any value to neither developer nor users.

Reply:
Ok

Reply:
Done